src/garage/tf/samplers/ray_batched_sampler.py

```diff
@@ -0,0 +1,71 @@
+"""Ray Sampler, for tensorflow algorithms.
+
+Currently the same as garage.samplers.RaySampler but includes
+support for Tensorflow sessions
+"""
+import ray
+import tensorflow as tf
+
+from garage.sampler import RaySampler, SamplerWorker
+
+
+class RaySamplerTF(RaySampler):
```

Comment:
@krzentner do you think the right pattern for this is inheritance? i guess it will be moot once we transition to TF2.0, and sessions don't really exist.

Reply:
I think inheritance is fine, in this cases.